Code/cmake/Modules/FindCairo.cmake

```diff
@@ -38,21 +38,32 @@
 # ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 FIND_PACKAGE(PkgConfig)
-PKG_CHECK_MODULES(PC_CAIRO cairo) # FIXME: After we require CMake 2.8.2 we can pass QUIET to this call.
+PKG_CHECK_MODULES(PC_CAIRO cairo QUIET)
 
 FIND_PATH(CAIRO_INCLUDE_DIRS
     NAMES cairo.h
-    HINTS ${PC_CAIRO_INCLUDEDIR}
+    HINTS ${CAIRO_INCLUDE_DIR}
+          ${PC_CAIRO_INCLUDEDIR}
           ${PC_CAIRO_INCLUDE_DIRS}
     PATH_SUFFIXES cairo
 )
 
 FIND_LIBRARY(CAIRO_LIBRARIES
     NAMES cairo
-    HINTS ${PC_CAIRO_LIBDIR}
+    HINTS ${CAIRO_LIBRARY_DIR}
+          ${PC_CAIRO_LIBDIR}
           ${PC_CAIRO_LIBRARY_DIRS}
 )
 
+IF (UNIX AND NOT APPLE)
+	PKG_CHECK_MODULES(PC_XAU xau QUIET)
+	FIND_LIBRARY(XAU_LIBRARIES
```

Comment:
Doesn't this reinforce an X dependency even if using cairo-nox?

Reply:
Hmmm, good point. I'll look into linking `Xau` only if linking with `cairo` fails due to missing `Xau`.